list/source/GridListImageItem.css

```diff
@@ -10,6 +10,9 @@
 .enyo-gridlist-imageitem img {
 	display: block;
 	width: 100%;
+	height: 100%;
+	background-size: 100% 100%;
```

Comment:
I agree on this. Removing **background-*** related CSS properties. I looked into preventive ways for distortion on images. We can use *max-height* property.